src/h5web/metadata-viewer/EntityInfo.tsx

```diff
@@ -24,17 +24,21 @@ function EntityInfo(props: Props): ReactElement {
         </tr>
       </thead>
       <tbody>
-        {'type' in entity && (
+        {(isDataset(entity) || isDatatype(entity)) && (
           <tr>
             <th scope="row">Type</th>
-            <td>{isBaseType(entity.type) ? entity.type.class : 'Complex'}</td>
+            <td>
+              {typeof entity.type === 'string'
+                ? entity.type
+                : entity.type.class}
+            </td>
```

Comment:
The term `Complex` was confusing and the condition was hiding useful information.